src/app/legacy/components/ScrollablePromo/index.jsx

```diff
@@ -72,22 +153,31 @@ const ScrollablePromo = ({ blocks, blockGroupIndex = null }) => {
     return null;
   }
 
-  const title =
-    blocks[0].type === 'title' &&
-    path(
-      ['0', 'model', 'blocks', '0', 'model', 'blocks', '0', 'model', 'text'],
-      blocks,
-    );
+  let title;
+  if (experimentVariant === 'A') {
+    title = translations.topStoriesTitle || 'Top Stories';
+  } else if (experimentVariant === 'B') {
+    title = mostRead.header || 'Most Read';
+  } else {
+    title =
+      blocks[0].type === 'title' &&
+      path(
+        ['0', 'model', 'blocks', '0', 'model', 'blocks', '0', 'model', 'text'],
+        blocks,
+      );
+  }
 
   const blocksWithoutTitle = blocks[0].type === 'title' ? tail(blocks) : blocks;
 
   const isSingleItem = blocksWithoutTitle.length === 1;
 
-  const ariaLabel = title && idSanitiser(title);
+  const ariaLabel = title && idSanitiser(`${title}`);
```

Comment:
Is this string interpolation needed?

Reply:
^ I think it can just be `const ariaLabel = title && idSanitiser(title);`

Reply:
Will change it to this. I used to add a number to the string in particular conditions to avoid an a11y violation, but I then solved the violation in a different way.